src/shared/Atlassian.Bitbucket.UI/ViewModels/OAuthViewModel.cs

```diff
@@ -30,17 +30,18 @@ public OAuthViewModel(IEnvironment environment)
 
         private void LearnMore()
         {
-            BrowserUtils.OpenDefaultBrowser(_environment, "https://confluence.atlassian.com/bitbucket/two-step-verification-777023203.html");
+            // 2FA is not supported on Server/DC so this prompt will never be seen outside of Bitbucket Cloud
+            BrowserUtils.OpenDefaultBrowser(_environment, BitbucketConstants.HelpUrls.TwoFactor);
         }
 
         private void ForgotPassword()
         {
-            BrowserUtils.OpenDefaultBrowser(_environment, "https://bitbucket.org/account/password/reset/");
+            BrowserUtils.OpenDefaultBrowser(_environment, BitbucketConstants.HelpUrls.PasswordReset);
         }
 
         private void SignUp()
         {
-            BrowserUtils.OpenDefaultBrowser(_environment, "https://bitbucket.org/account/signup/");
+            BrowserUtils.OpenDefaultBrowser(_environment, BitbucketConstants.HelpUrls.SignUp);
         }
```

Comment:
Another thought occurred to me - is it possible to hide these links in a DC/Server environment where they're not applicable?

Reply:
It would be possible, but the entire OAuth dialog is not applicable to Server/DC in the first place! If we're showing this dialog it must be for Cloud.

Reply:
Got it!